source/browser.ts

```diff
@@ -123,6 +123,10 @@ ipc.on('insert-emoji', async () => {
 	emojiElement.click();
 });
 
+ipc.on('insert-files', () => {
+	document.querySelector<HTMLElement>('._7oam input[type="file"]')!.click();
```

Comment:
`. _7oam` doesn't exist in the old messenger UI. There, the path would probably be:

```css
._5vn8 + input[type="file"]
```

Reply:
Oh, I wasn't aware that the old Messenger UI still exists.

Is there any way I can access the old UI? So I can test that as well.

Reply:
Some people have reported that using a different browser makes the Messenger load the other UI version for them, but there's no reliable way to choose the UI version.

Anyway, `._5vn8 + input[type="file"]` does work in my (old UI) version (`.5vn8` is the "Add files" icon, and the input element directly follows it).

Reply:
Okay, that's fine, we can support both the old version and the new version.

What do you think about something like this?
```suggestion
	// old UI
	document.querySelector<HTMLElement>('._5vn8 + input[type="file"]')!.click();
	// new UI
	document.querySelector<HTMLElement>('._7oam input[type="file"]')!.click();
```

Reply:
This is done in https://github.com/sindresorhus/caprine/pull/1014/commits/b34aaf33d1157abb281595b6907bafc49e069e3f.